### PR TITLE
fix #306603: Measure number vertical offset interpreted incorrectly

### PR DESCRIPTION
**Pull request by Marr11317**
Resolves: https://musescore.org/en/node/306603

See [this comment](https://github.com/musescore/MuseScore/pull/5817#discussion_r438798409) for details

- [x] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/Documentation/blob/master/WorkflowAndGuidelines/CodeGuidelines.md)
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [x] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

File: libmscore/style.cpp [hunk 1]
```diff
@@ -834,7 +834,8 @@ static const StyleType styleTypes[] {
       { Sid::measureNumberFontSpatiumDependent, "measureNumberFontSpatiumDependent", true },
       { Sid::measureNumberFontStyle,        "measureNumberFontStyle",       int(FontStyle::Normal) },
       { Sid::measureNumberColor,            "measureNumberColor",           QColor(0, 0, 0, 255) },
-      { Sid::measureNumberOffset,           "measureNumberOffset",          QPointF(0.0, 0.0) },
+      { Sid::measureNumberPosAbove,         "measureNumberOffset",          QPointF(0.0, -2.0) }, // This measureNumberOffset cannot be renamed to measureNumberPosAbove for backward compatibility
```
Comment (MarcSabatella) [inline libmscore/style.cpp:837]: FWIW, you probably *could* rename it and add code when reading that tag to actually set the other, and same on writing.  But, this is certainly the simpler solution.

**Comment by mattmcclinch** [comment]
Why such a big change to `editstyle.ui`? Just add the lines you need and maybe change some row numbers.

**Comment by MarcSabatella** [comment]
Tested and works as expected, thanks!

**Comment by Marr11317** [comment]
Marked as wip because the offset for above is still used when importing measure numbers with pos below... 
Working on it right now.

@mattmcclinch it's true that there's more line changes than needed in editstyle.ui, but that's because of qtcreator, and by using .ui files, we accept that qtcreator saves as he likes the file. I could modify it, but it would take more time than it should and for a little bit more than 100 lines, I don't think it's needed.

File: libmscore/style.cpp (hunk 1 above)
Comment (MarcSabatella) [reply libmscore/style.cpp:837]: To me not worth the trouble of doing it differently than this.

**Comment by mattmcclinch** [comment]
I went ahead and cleaned up your changes to `editstyle.ui`. The result is not only a [much nicer diff](https://github.com/mattmcclinch/MuseScore/commit/fbd34ceba45686d47e8f86de169ed1733d0a0df5#diff-b59d46515d2677f362a2ad7ebb2aa9f4), but also a slightly improved layout, in my opinion. Here are some screen shots for comparison.

Prior to this PR:
old

Your proposed changes:
yours

My proposed changes:
mine

**Comment by MarcSabatella** [comment]
Definitely nicer, thanks for that, @mattmcclinch .

It's "OK" as is but would be much better with that change.

**Comment by Marr11317** [comment]
Thank you! You are right, it is nicer that way.

**Comment by Marr11317** [comment]
@mattmcclinch I rebased with your version. Thanks again!

**Pull request merged**